### PR TITLE
feat: Import Assets Text Style Catalog

### PR DESCRIPTION
**Pull request by lucianoschillagi**

### CODE REVIEW

**Review by stavares843 — APPROVED** [review]

**Review by Power186 — COMMENTED** [review]
Was it intentional to change the deployment target from 13.6 to 13.3?

Was it intentional to change the development team?

**Review by Power186 — APPROVED** [review]
Looks good, but will checkout deployment target change and dev team change

File: FightPandemics.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -438,7 +490,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.6;
```
Comment (Power186) [inline FightPandemics.xcodeproj/project.pbxproj:441]: Should not be changed, merge conflict

File: FightPandemics.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -492,7 +544,7 @@
...
```
Comment (Power186) [inline FightPandemics.xcodeproj/project.pbxproj:495]: Merge conflict

File: FightPandemics.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -528,7 +580,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "\"FightPandemics/Preview Content\"";
-				DEVELOPMENT_TEAM = Q7KJCMM3NN;
```
Comment (Power186) [inline FightPandemics.xcodeproj/project.pbxproj:531]: Merge conflict

**Review by Power186 — CHANGES_REQUESTED** [review]

**Comment by lucianoschillagi** [comment]
Ok, thanks. I will do this changes.

**Comment by lucianoschillagi** [comment]
I am using **Xcode 11.4** and my max iOS Development target is **13.4.**
Suggestions?

**Comment by Power186** [comment]
> I am using **Xcode 11.4** and my max iOS Development target is **13.4.**
> Suggestions?

Xcode is updated with the updates from Catalina.  My suggestion for now would be to update to the latest Catalina.  Another option, would be downloading the repo from .zip again, opening and making the the import asset changes without touching the deployment target or version..

I'll look more into it today for workarounds other than these two options.  Please let me know if either worked.

**Pull request closed without merging**